scripts/rff/01_aggregate_PDSI.R

```diff
@@ -0,0 +1,214 @@
+library(ncdf4) 
```

Comment:
Could these be moved inside climate_wildfire_smoke directory?

Also, could we please add a README.md in that directory with a brief overview of the scripts?

The goal is for people interested in understanding / reproducing the data from the source to be able to do it.

Reply:
Scripts moved and added readme